### PR TITLE
Vectorizer and Label Encoder Changes 

### PR DESCRIPTION
**Pull request by saanikat**

### CODE REVIEW

File: bedms/attr_standardizer.py [hunk 1]
```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
...
+    def _load_model(self) -> tuple[nn.Module, object, object]:
         """
+<<<<<<< HEAD:bedms/attr_standardizer.py
+        Calls function to load the model from HuggingFace repository
+          load vectorizer and label encoder and sets to eval().
+=======
         Calls function to load the model from HuggingFace repository and sets to eval().
-
+>>>>>>> b13f0a3f927e52b5ccb9ec6f36064f19a1f5af9e:attribute_standardizer/attr_standardizer.py
         :return nn.Module: Loaded Neural Network Model.
```
Comment (khoroshevskyi) [inline bedms/attr_standardizer.py:110]: There were conflicts, so something is wrong here

File: bedms/attr_standardizer.py [hunk 2]
```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
...
+        :return object: The scikit learn vectorizer for bag of words encoding.
+        :return object: Label encoder object for the labels (y).
```
Comment (khoroshevskyi) [inline bedms/attr_standardizer.py:112]: Update docstring

**Review by khoroshevskyi — APPROVED** [review]
Looks good to me, just one comment

File: bedms/attr_standardizer.py [hunk 3]
```diff
@@ -28,13 +38,17 @@
     fetch_from_pephub,
     get_any_pep,
     load_from_huggingface,
+    hf_hub_download,
```
Comment (nleroy917) [inline bedms/attr_standardizer.py:41]: Isnt this from the `huggingface_hub` package? Shouldnt we import directly there? Or is this some sort of shadowed wrapper?

File: bedms/attr_standardizer.py [hunk 4]
```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
...
-    def _load_model(self) -> nn.Module:
+    def _load_model(self) -> tuple[nn.Module, object, object]:
         """
+<<<<<<< HEAD:bedms/attr_standardizer.py
+        Calls function to load the model from HuggingFace repository
+          load vectorizer and label encoder and sets to eval().
+=======
         Calls function to load the model from HuggingFace repository and sets to eval().
-
+>>>>>>> b13f0a3f927e52b5ccb9ec6f36064f19a1f5af9e:attribute_standardizer/attr_standardizer.py
```
Comment (nleroy917) [inline bedms/attr_standardizer.py:109]: Yeah this needs to be fixed... Looks like we committed and pushed mid-merge

File: bedms/attr_standardizer.py [hunk 5]
```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
...
-            raise ValueError(
-                f"Schema not available: {self.schema}. Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
+    
+        raise ValueError(
+                f"Schema not available: {self.schema}."
+                "Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
             )
 
-    def _load_model(self) -> nn.Module:
+    def _load_model(self) -> tuple[nn.Module, object, object]:
```
Comment (nleroy917) [inline bedms/attr_standardizer.py:102]: Is this syntax (`tuple[...]`) supported? Do we not need `typing.Tuple`?

**Review by nleroy917 — CHANGES_REQUESTED** [review]
Just need to get rid of those merge conflicts...

File: bedms/attr_standardizer.py (hunk 5 above)
Comment (khoroshevskyi) [reply bedms/attr_standardizer.py:103]: yeah, we need

Comment (nleroy917) [reply bedms/attr_standardizer.py:103]: correct me if I am wrong here... in my mind `tuple` is ok for python >= 3.10

`typing.Tuple` is needed for anything less?

File: bedms/attr_standardizer.py (hunk 4 above)
Comment (saanikat) [reply bedms/attr_standardizer.py]: Yes, updated that!
